content/md/what-is/what-is-an-attribute.md

```diff
@@ -5,63 +5,114 @@ title: What is an **attribute**?
+- **reference entity simple link** attribute _(Entreprise Edition only)_: allows enriching common data related to products with a rich content (text, images...), more complex than just a code and labels. 
+- **reference entity multiple links** attribute _(Entreprise Edition only)_: the same as above but allowing you to manage multiple links
+- **reference data simple select** attribute: allows you to manage any kind of data that has its own properties, as a single-choice select. 
+- **reference data multi select** attribute: allows you to manage any kind of data that has its own properties, as a multi-choice select. 
+
+:::warning
+You can have only one attribute `identifier`, but you can have several attributes from the other types. If you need more identifier fields, use a text field.
+:::
 
 :::info
 The only mandatory attribute is the **Identifier attribute type**. You first need to create an identifier attribute type before creating products.    
-You can only have one **identifier attribute type** in your PIM.
 :::
 
+:::tips
+Options for simple and multi select attributes can be translated in any enabled locale in the PIM.
+:::
 
 :::tips
-To store GTIN, EAN13, UPC… in your PIM, use the **text attribute type** and use a regular Expression to make sure that your codes will contain the exact number of digits required.  
-For instance, to store an EAN13 in Akeneo, which contains 13 digits, add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
+To store GTIN, EAN, UPC… in your PIM, use the **text** attribute type and use a regular expression to control the field input. 
+For instance, to store an EAN13 in Akeneo, you can easily add a validation rule based on the following regular expression: `/^[0-9]{13}$/`
 :::
 
+## Reference entities or reference data?
+
+### Differences between the two attribute types 
+
+**Reference entities are an Enterprise Edition feature added in the 3.0 version**. To learn more about Reference entities, read our dedicated article [What is a Reference Entity?](what-about-reference-entities.html).    
+Whereas reference data are a native Community and Enterprise editions feature which implies a custom development using the [Custom Entity Bundle](https://github.com/akeneo-labs/CustomEntityBundle) to create the entities.
+
+**If you are starting a new project from version 3.0, we strongly recommand you to use the [Reference entities](manage-reference-entities.html#create-a-reference-entity).**
+
+### I was using Reference data, can I transform them into Reference entities?
+
+Yes! If you were using Reference Data in a previous PIM version, you can migrate your Reference data thanks to the [`CSVToReferenceEntity`](https://github.com/akeneo/CsvToReferenceEntity) tool. 
+`CSVToReferenceEntity` will allow you to migrate data from the Custom Entity Bundle and transform them into Reference Entities records. 
```

Comment:
Add the tool link: https://github.com/akeneo/CsvToReferenceEntity

Reply:
je l'ai mis juste au dessus :)

Reply:
ah yes